diff --git a/drivers/media/platform/mxc/capture/ov5640_mipi_v2.c b/drivers/media/platform/mxc/capture/ov5640_mipi_v2.c
index eac8b12360dc..fbe15e6c54a7 100644
--- a/drivers/media/platform/mxc/capture/ov5640_mipi_v2.c
+++ b/drivers/media/platform/mxc/capture/ov5640_mipi_v2.c
@@ -1807,7 +1807,7 @@ static int ov5640_probe(struct i2c_client *client,
 		return -ENODEV;
 	}
 	retval = ov5640_read_reg(sensor, OV5640_CHIP_ID_LOW_BYTE, &chip_id_low);
-	if (retval < 0 || chip_id_low != 0x40) {
+	if (retval < 0 || chip_id_low != 0x45) {
 		dev_warn(dev, "Camera is not found\n");
 		ov5640_regualtor_disable();
 		clk_disable_unprepare(sensor->sensor_clk);
